contractcourt/channel_arbitrator.go

```diff
@@ -1323,20 +1337,17 @@ func (c *ChannelArbitrator) sweepAnchors(anchors *lnwallet.AnchorResolutions,
 		)
 
 		// Sweep anchor output with a confirmation target fee
-		// preference. Because this is a cpfp-operation, the anchor will
-		// only be attempted to sweep when the current fee estimate for
-		// the confirmation target exceeds the commit fee rate.
-		//
-		// Also signal that this is a force sweep, so that the anchor
-		// will be swept even if it isn't economical purely based on the
-		// anchor value.
+		// preference. Because this is a cpfp-operation, the anchor
```

Comment:
Yeah what we want to start to do there is actually use `testmempoolaccept` before we broadcast things. So we become less "spray and pray" in general.

Reply:
> It would be great that we also include a Ratio Check in the Fee-Estimation, meaning that if we have a deadline sweep we do only sweep it if the new Fee is at least greater that 5% (or similar) of the original value.

Cool yeah this will be handled by the incoming fee bumper.